src/components/Navbar.js

```diff
@@ -1,38 +1,40 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 import { FaUser } from 'react-icons/fa';
+import './styles/Navbar.css';
 
 function Navbar() {
   return (
-    <nav>
+    <nav className="navbar">
       <div className="logo-container">
         <a className="logo" href="/">
-          BOOKSTORE
+          Bookstore
         </a>
-        <ul className="links">
-          {[
-            {
-              id: 1,
-              path: '/',
-              name: 'BOOKS',
-            },
-            {
-              id: 2,
-              path: '/categories',
-              name: 'CATEGORIES',
-            },
-          ].map((link) => (
-            <li key={link.id}>
-              <NavLink className="nav-link" to={link.path}>
-                {link.name}
-              </NavLink>
-            </li>
-          ))}
-        </ul>
-
-        <div className="person">
+      </div>
+      <ul className="links">
+        {[
+          {
+            id: 1,
+            path: '/',
+            name: 'BOOKS',
+          },
+          {
+            id: 2,
+            path: '/categories',
+            name: 'CATEGORIES',
+          },
+        ].map((link) => (
+          <li key={link.id}>
+            <NavLink className="nav-link" to={link.path}>
+              {link.name}
+            </NavLink>
+          </li>
+        ))}
+      </ul>
```

Comment:
- [ ] @siddghosh108 Great work styling your app, however I noticed a slight difference in styling between your navbar items and the given design, check the comparison below:


Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/siddghosh108/Book-Store-Redux/assets/61408860/ce6509ef-d70d-4aa6-ac23-6ba4d56531d4)  | ![image](https://github.com/siddghosh108/Book-Store-Redux/assets/61408860/bd26235e-9363-4d71-a289-860821cb61a8)

Reply:
Now the font is thin and sleek just like the way you wanted
![Uploading image.png…]()